pages/linux/cpufreq-set.md

```diff
@@ -0,0 +1,20 @@
+# cpufreq-set
+
+> A tool to modify CPU frequency settings.
+> Requires root privileges.
+
+- Set the CPU frequency policy of CPU 1 to "userspace":
+
+`sudo cpufreq-set -c {{1}} -g {{userspace}}`
+
+- Set the current minimum CPU frequency of CPU 1, value range between the output of command `cpufreq-info -l`:
```

Comment:
My only concern is that the "value range between the output of command" is repeated 3 times. Is there any way we can just mention it once ? Maybe put this as the subheading and remove the "Requires root privileges" part since `sudo` is already there in the example ?